node/pkg/logscribe/logprocessor/logprocessor.go

```diff
@@ -201,28 +201,33 @@ func (p *LogProcessor) CreateGithubIssue(ctx context.Context, processedLogs []Lo
 
 	issueCount := 0
 	processedLogHashes := [][]interface{}{}
+	nonprocessedLogs := []LogInsertModelWithCount{}
```

Comment:
nonprocessedLogs seems to be a bit confusing, at first glance I couldn't understand if it a non processed log within the logic or logs which is previously not processed.

how about renaming it as `excludedLogs` or `duplicateLogs`?